wsd/annotate/app.py

```diff
@@ -75,8 +72,8 @@ def get_entries():
     return lindict.search(lemma)
 
 
-def on_load(e):
```

Comment:
`on_load` need to have an argument to avoid an error on launch:

<img width="870" alt="Screenshot 2025-03-17 at 7 30 09" src="https://github.com/user-attachments/assets/01dd0fc4-26e3-456e-afa8-8dd28770caea" />